package.json

```diff
@@ -22,12 +24,15 @@
     "url": "git@github.com:bjrmatos/jsreport-jade.git"
   },
   "dependencies": {
+    "in-publish": "2.0.0",
```

Comment:
> Apparently npm install runs prepublish script (npm/npm#3059) 
> That is why we use https://github.com/iarna/in-publish to skip the prepublish script during npm install

yes, this is correct, an annoying behaviour but solved with `in-publish` :)



> If I run npm install, I don't have devDependencies installed, for sure not for the nested dependencies, so I guessed I need to put in-publish to dependencies. As far I remember I even got error in travis: in-publish not found before I moved it to devDepndencies.

hmm I have not had this problem, even with a fresh install npm seems to resolve the `prepublish` hooks correctly (at least in OSX), the only problem that i see is if you run `npm install --production` with `in-publish` in `devDependencies`